pkg/services/sqlstore/migrator/migrator.go

```diff
@@ -185,9 +185,28 @@ func (mg *Migrator) run() (err error) {
 	return mg.DBEngine.Sync2()
 }
 
-func (mg *Migrator) exec(m Migration, sess *xorm.Session) error {
+func (mg *Migrator) exec(m Migration, sess *xorm.Session) (bool, error) {
```

Comment:
The meaning of the result is not very clear. It would be great to have some docs for the method